src/celldega/nbhd/test_hextile.py

```diff
@@ -0,0 +1,14 @@
+from . import *
+from ..pre.merge_segmentations import *
+
+def test_hextile():
+
+    gdf_dummy_trx = generate_random_points_gdf(n_points=50, x_range=(0, 100), y_range=(0, 100))
+    dummy_img = generate_dummy_image(width=100, height=100)
```

Comment:
we can drop the generate_dummy_image method since we are just making an image in order to calculate the area of the image and just calculate the intended area instead.